semgrep_output_v1.atd

```diff
@@ -1472,6 +1472,8 @@ type engine_configuration = {
     ~generic_slow_rollout: bool;
     (* from 1.63.0 *)
     ?historical_config: historical_configuration option;
+    (* for customers to use the app to configure fail open options *)
+    ~always_fail_open: bool;
```

Comment:
I'm actually unclear on this, perhaps @emjin can chime in. Is there a difference between `~` and `?` ?. I can't really find anything in the docs other than https://atd.readthedocs.io/en/latest/atdgen-tutorial.html#smooth-protocol-upgrades. 

Seems like this should be a safe upgrade and shouldn't break an existing scans when merged in (looks like we default to false if it's not defined).

Reply:
`~` is a field with a default value, `?` is an optional field that can be None or Some x